app/Resources/views/job-advert.html.twig

```diff
@@ -2,6 +2,16 @@
 
 {% block title %}{{ title|striptags }} | Jobs{% endblock %}
 
+{% block meta %}
+
+    {{ parent() }}
+
+    {% if date(jobAdvert.closingDate) < date() %}
```

Comment:
it's the Twig `date()` rather than PHP's `date()`, but doesn't it need a `format` argument?

Reply:
Nope, it's actually `new DateTime()`. (You might be thinking of the `date` filter.)